app/src/androidTest/java/com/duckduckgo/app/browser/BrowserWebViewClientTest.kt

```diff
@@ -207,13 +223,49 @@ class BrowserWebViewClientTest {
         verify(emailInjector).injectEmailAutofillJs(webView, null)
     }
 
+    @UiThreadTest
+    @Test
+    fun whenOnPageFinishedCalledThenFlushCookies() = runBlocking {
+        testee.onPageFinished(webView, null)
+        withContext(Dispatchers.IO) {
```

Comment:
I've updated the `cookieManager.flush()` call in `BrowserWebViewClient` to use `dispatcherProvider.io()` instead of `Dispatchers.IO`. `runBlocking` and `withContext` have been removed from the test 👍